keadm/cmd/keadm/app/cmd/config.go

```diff
@@ -0,0 +1,229 @@
+/*
+Copyright 2022 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/kubeedge/kubeedge/common/constants"
+	cmdcommon "github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/common"
+	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/util"
+)
+
+// PullImageRetry represent pulling images retry times
+const PullImageRetry = 5
+
+// Configuration represent keadm config options
+type Configuration struct {
+	// eg. v1.9.0
+	KubeEdgeVersion string
+	// eg. kubeedge
+	ImageRepository string
+	// eg. cloud/edge
+	Part string
+}
+
+func newDefaultConfiguration() *Configuration {
+	var latestVersion string
+	var kubeedgeVersion string
+	for i := 0; i < util.RetryTimes; i++ {
+		version, err := util.GetLatestVersion()
+		if err != nil {
+			fmt.Println("Failed to get the latest KubeEdge release version, error: ", err)
+			continue
+		}
+		if len(version) > 0 {
+			kubeedgeVersion = strings.TrimPrefix(version, "v")
+			latestVersion = version
+			break
+		}
+	}
+	if len(latestVersion) == 0 {
+		kubeedgeVersion = cmdcommon.DefaultKubeEdgeVersion
+		fmt.Println("Failed to get the latest KubeEdge release version, will use default version: ", kubeedgeVersion)
+	}
```

Comment:
The latest version obtained here is more suitable for prompting, 
and it is better to use the current version of `keadm` as the default version.

Works like `kubeadm`:
```
[root@master-7-1 ~]# kubeadm config images list
I0111 15:17:59.444108   14703 version.go:251] remote version is much newer: v1.23.1; falling back to: stable-1.20
W0111 15:18:09.444598   14703 version.go:101] could not fetch a Kubernetes version from the internet: unable to get URL "https://dl.k8s.io/release/stable-1.20.txt": Get "https://dl.k8s.io/release/stable-1.20.txt": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
W0111 15:18:09.444641   14703 version.go:102] falling back to the local client version: v1.20.0
k8s.gcr.io/kube-apiserver:v1.20.0
k8s.gcr.io/kube-controller-manager:v1.20.0
k8s.gcr.io/kube-scheduler:v1.20.0
k8s.gcr.io/kube-proxy:v1.20.0
k8s.gcr.io/pause:3.2
k8s.gcr.io/etcd:3.4.13-0
k8s.gcr.io/coredns:1.7.0
```

Reply:
OK, will submit another PR to fix it.

Reply:
now, previous `keadm init` logic is also using the latest version, need fixing both in `keadm init` cmd and in `keadm beta init` cmd